README.md

```diff
@@ -117,8 +137,20 @@ If you find **any vulnerabilities**, please raise an issue! ⚠️
 
 ## Purpose and Ethical Use ⚖️
 
+
+## Contributors
```

Comment:
I think the contributors images at the left of the page is enough.
Adding the contributors again to the Readme seems useless no?

Reply:
Alright I will resolve this